src/backend/marsha/core/utils/lti_select_utils.py

```diff
@@ -0,0 +1,32 @@
+"""Utils for working with lti_select modules."""
+from functools import cache
+
+import django
+from django.utils.module_loading import import_string
+
+
+@cache
+def get_lti_select_resources():
+    """Look for all availeble lti select resources."""
+
+    result = []
+    marsha_apps = [
+        app
+        for app in django.apps.apps.app_configs.values()
+        if app.name.startswith("marsha.")
+    ]
+    for marsha_app in marsha_apps:
```

Comment:
```suggestion
    for app in django.apps.apps.app_configs.values():
        if not app.name.startswith("marsha."):
            continue
```
I find it more readable.

Reply:
What ? You don't like list comprehension ? I though that was the holy grail for pythonist

Reply:
:D
In this case, as we are looping 2 times, I find the first one useless.